docs/DeveloperGuide.md

```diff
@@ -532,133 +532,183 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1. User requests to add a friend and specifies a `FRIEND_ID`.
+1. User requests to add a friend and specifies a `FRIEND_ID` and optionally specifies a `NAME`.
 2. gitGud adds a friend into the friends list and informs user of successful addition of friend.
 
     Use case ends.
 
 **Extensions** 
 
 * 1a. gitGud detects that the `FRIEND_ID` provided already exists and is not unique.
-    * 1a1. gitGud informs user that it has failed to add a friend.<br>
-      1a2. User makes another request to add a friend with a different `FRIEND_ID`.<br>
-      Use case continues from step 2.
+    * 1a1. gitGud informs user that it has failed to add a friend.
+    * 1a2. User makes another request to add a friend with a different `FRIEND_ID`.
+    * Use case continues from step 2.
 
-**Use case: UC02 - Delete a friend**
+**Use case: UC02 - Edit a friend**
 
 **MSS**
 
-1. User requests to delete a friend using the friend's `FRIEND_ID`
-2. gitGud deletes the person
+1. User requests to edit a friend and specifies a `FRIEND_ID` and new `NAME` to update.  
+2. gitGud updates friend and informs user of successful edit of friend.  
 
-    Use case ends.
+   Use case ends.  
 
 **Extensions**
 
-* 2a. The given `FRIEND_ID` is invalid.
+* 1a. gitGud detects that the `FRIEND_ID` does not exist.  
+    * 1a1. gitGud informs user that it has no friend was found with specified `FRIEND_ID`.  
+      1a2. User makes another request to edit a friend with a different `FRIEND_ID`.  
```

Comment:
Maybe here you can say something like:

```suggestion
      1a2. User makes another request to edit a friend with a different `FRIEND_ID` which exists in gitGud.  
```

Because if it doesn't exist then it won't continue from Step 2

Reply:
I agree, this is clearer (same for the other points below regarding existing friend/game).